test/com/ansorgit/plugins/bash/lang/psi/util/BashIdentifierUtilTest.java

```diff
@@ -59,7 +59,7 @@ public void testValidIdentifier() {
 
         Assert.assertFalse(isValidIdentifier("1a"));
         Assert.assertFalse(isValidIdentifier("a$a"));
-        Assert.assertFalse(isValidIdentifier("a-a"));
+//        Assert.assertFalse(isValidIdentifier("a-a"));
```

Comment:
Please, take a look at this change. Maybe we have to allow use "-" sign in identifiers? What do you think?